webpack.common.js

```diff
@@ -52,6 +56,10 @@ module.exports = {
           fallback: 'style-loader',
           use: 'css-loader!sass-loader'
         })
+      }, {
+        test: /\.(png|svg|jpg|gif)$/,
+        use: ['file-loader'],
+        include: path.join(__dirname, 'src/assets')
```

Comment:
Since the `src` folder is resolved, this should be `path.join(paths.src, 'assets')` if I am correct.